examples/kepler-integration/src/components/export-video/parse-set-camera-type.js

```diff
@@ -52,16 +52,16 @@ export function parseSetCameraType(strCameraType, viewState) {
   if (setChecker.has(match[0])) {
     if (match[0] === 'East') {
       // TODO Temporary solution to catch this branch to master. Doesn't work for "East to North" for example if option allows in future
-      const translatedPoly = transformTranslate(turfPoint, 10000, 270);
+      const translatedPoly = transformTranslate(turfPoint, 10, 270);
```

Comment:
Oh nice, does this fix a bug?

Reply:
that field is for KMs to move. For the scale of the world 10k made sense. Not so much when we're focused on a single city.

Reply:
Sounds good. We should think of a pleasant dynamic rule for this that changes with zoom level.

Reply:
Agreed. Will add it as a task